go/client/cmd_ctl_start.go

```diff
@@ -4,33 +4,73 @@
 package client
 
 import (
+	"fmt"
+
 	"github.com/keybase/cli"
+	"github.com/keybase/client/go/install"
 	"github.com/keybase/client/go/libcmdline"
 	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
 )
 
 func NewCmdCtlStart(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 	return cli.Command{
 		Name:  "start",
 		Usage: "Start the background keybase service",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "t, type",
+				Usage: fmt.Sprintf("Service type (%s, %s), default is %q", keybase1.ForkType_AUTO.String(), keybase1.ForkType_LAUNCHD.String(), defaultForkType.String()),
+			},
+		},
 		Action: func(c *cli.Context) {
-			cl.ChooseCommand(&CmdCtlStart{libkb.NewContextified(g)}, "start", c)
-			cl.SetForkCmd(libcmdline.ForceFork)
-			cl.SetNoStandalone()
+			cl.ChooseCommand(NewCmdCtlStartRunner(g), "start", c)
+			cl.SetForkCmd(libcmdline.NoFork)
+			cl.SetLogForward(libcmdline.LogForwardNone)
 		},
 	}
 }
 
 type CmdCtlStart struct {
 	libkb.Contextified
+	forkType keybase1.ForkType
+}
+
+func NewCmdCtlStartRunner(g *libkb.GlobalContext) *CmdCtlStart {
+	return &CmdCtlStart{Contextified: libkb.NewContextified(g)}
 }
 
 func (s *CmdCtlStart) ParseArgv(ctx *cli.Context) error {
+	forkTypeStr := ctx.String("type")
```

Comment:
I'd be very careful changing anything here, it would be very easy to break something.  Also, it would be great to fix the problem that we infer OSX launchd fork via `KEYBASE_LABEL` environment variable, which feel very hacky